src/psyclone/psyir/nodes/loop.py

```diff
@@ -82,6 +82,10 @@ class Loop(Statement):
     _text_name = "Loop"
     _colour = "red"
 
+    # Set of rules that give the loop a certain loop_type name by inspecting
+    # their variable and start/stop expressions.
```

Comment:
Yes, but I made it more generic now with:
`# Set of rules that give the loop a certain loop_type name by inspecting it`
as this can potentially be extended